ui/src/components/discovery/Discovery.js

```diff
@@ -6,7 +6,7 @@ import L from 'leaflet'
 import prepareQuery from './/queryBuilder'
 import OwnMap, { ref, ref2 } from "./Map"
 import ResultList from './resultList'
-import config from '../../helpers/config';
+//import config from '../../helpers/config';
```

Comment:
If something is not needed anymore, please remove, or add a further comment why this might be needed in the future.